eth_validator_watcher/next_blocks_proposal.py

```diff
@@ -20,6 +20,7 @@ def process_future_blocks_proposal(
     our_pubkeys: set[str],
     slot: int,
     is_new_epoch: bool,
+    slots_per_epoch: int = NB_SLOT_PER_EPOCH,
```

Comment:
nit: It'd be better to not default to something here and unconditionally use the value from the spec ; happy to do a pass on top of your PR to force this parameter :)